com.unity.cinemachine.hdrptests/Tests/Runtime/CheckPhysicalCameraProperties.cs

```diff
@@ -0,0 +1,117 @@
+using System.Collections;
+using Cinemachine;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Tests.HDRP.Runtime
+{
+    [TestFixture]
+    public class CheckPhysicalCameraProperties : CinemachineHDRPFixtureBase
+    {
+        Camera m_Cam;
+        CmCamera m_CmCamera;
+
+        [SetUp]
+        public override void SetUp()
+        {
+            base.SetUp();
+            
+            m_Cam = CreateGameObject("MainCamera", typeof(Camera), typeof(CinemachineBrain)).GetComponent<Camera>();
+            m_Cam.usePhysicalProperties = true;
+            
+            var vcamHolder = CreateGameObject("CM Vcam", typeof(CmCamera), typeof(CinemachineConfiner2D));
+            m_CmCamera = vcamHolder.GetComponent<CmCamera>();
+        }
+
+        [TearDown]
+        public override void TearDown()
+        {
+            base.TearDown();
+        }
+        
+        [UnityTest]
+        public IEnumerator IsPhysical()
+        {
+            yield return null;
+            
+            Assert.True(m_Cam.usePhysicalProperties);
+            Assert.True(m_CmCamera.Lens.IsPhysicalCamera);
+        }
+
+        [UnityTest]
+        public IEnumerator CheckAll()
+        {
+            // Check that initial lens state of cmCamera is equal to Camera's lens
+            yield return null;
+            {
+                var state = m_CmCamera.State;
+                Assert.That(m_Cam.sensorSize, Is.EqualTo(state.Lens.SensorSize));
```

Comment:
Lots of repeated code in checking lens properties, susceptible to error.  Can checking lens properties be moved to a function?